kuma/api/v1/tests/test_bookmarks.py

```diff
@@ -46,8 +52,7 @@ def test_is_bookmarked_signed_in(user_client, wiki_user, settings):
 
 @pytest.mark.django_db
 def test_toggle_bookmarked(user_client, wiki_user, mock_requests, settings):
-    # TEMPORARY until all things auth + subscription come together.
-    settings.FAKE_USER_SUBSCRIBER_NUMBER = 1234
+    UserProfile.objects.create(user=wiki_user, is_subscriber=timezone.now())
```

Comment:
I went one step further. Instead of an extension of `wiki_user` I created an extension of `user_client` called `subscriber_client`. 
Now, there are no fixtures arguments in any of the `def test_*(...)` functions in `test_bookmarks.py` that is not used.